src/codemodder/codemods/utils.py

```diff
@@ -113,6 +113,10 @@ def is_django_settings_file(file_path: Path):
     return False
 
 
+def is_setup_py_file(file_path: Path):
```

Comment:
I think this is probably pretty specific to the setup.py codemod and maybe doesn't belong here.

Reply:
In an effort to generalize and since I don't quite yet know how the repo manager will work, I think it's worth keeping it here for now.